### ISSUE TITLE
Can't retrieve fps where species contains "-"

### ISSUE THREAD

**Issue by joe-pitt** [openghg/openghg#1003]
### What happened?

Trying to retrieve fps for hfos (e.g. hfo-1234yf) from the shared store - looks like it is discarding the "-" character when it searches

### What did you expect to happen?

Fp should be retrieved

### Minimal Complete Verifiable Example

```Python
fp = retrieve.get_footprint(site="mhd",
                            domain="europe",
                            start_date="2023-01-01",
                            end_date="2023-01-08",
                            species="hfo-1234yf",
                            met_model="ukv",
                            store="shared")
```

### Relevant log output

```Python
---------------------------------------------------------------------------
SearchError                               Traceback (most recent call last)
Cell In[2], line 1
----> 1 fp = retrieve.get_footprint(site="mhd",
      2                             domain="europe",
      3                             start_date="2023-01-01",
      4                             end_date="2023-01-08",
      5                             species="hfo-1234yf",
      6                             met_model="ukv",
      7                             store="shared")

File ~/openghg/openghg/retrieve/_access.py:650, in get_footprint(site, domain, inlet, height, model, start_date, end_date, species, **kwargs)
    647 inlet = format_inlet(inlet)
    648 height = format_inlet(height)
--> 650 fp_data = _get_generic(
    651     site=site,
    652     domain=domain,
    653     inlet=inlet,
    654     height=height,
    655     model=model,
    656     start_date=start_date,
    657     end_date=end_date,
    658     species=species,
    659     data_type="footprints",
    660     **kwargs,
    661 )
    663 return FootprintData(data=fp_data.data, metadata=fp_data.metadata)

File ~/openghg/openghg/retrieve/_access.py:53, in _get_generic(sort, elevate_inlets, ambig_check_params, **kwargs)
     51     err_msg = f"Unable to find results for {keyword_string}"
     52     logger.exception(err_msg)
---> 53     raise SearchError(err_msg)
     55 if sort:
     56     logger.warning(
     57         "Sorting results by time has been moved to the data method of ObsData, please use that instead."
     58     )

SearchError: Unable to find results for site='mhd', domain='europe', start_date='2023-01-01', end_date='2023-01-08', species='hfo1234yf', data_type='footprints', met_model='ukv', store='shared'
```

### Environment

python v3.12.3
Devel branch (commit a1a63a25b557d2f89f461990fa8ce4f879e82382)

**Comment by SutarPrasad** [comment]
Gets closed by #969 as species will be stored by their synonyms from openghg_defs

**Comment by SutarPrasad** [comment]
closed through https://github.com/openghg/openghg/pull/984
